[2022-09-23 14:36:21,136] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: kag_to_gcs_dag.cleanup_list_task manual__2022-09-23T14:31:12.806772+00:00 [queued]>
[2022-09-23 14:36:21,145] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: kag_to_gcs_dag.cleanup_list_task manual__2022-09-23T14:31:12.806772+00:00 [queued]>
[2022-09-23 14:36:21,145] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-23 14:36:21,145] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-09-23 14:36:21,146] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-23 14:36:21,156] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): cleanup_list_task> on 2022-09-23 14:31:12.806772+00:00
[2022-09-23 14:36:21,162] {standard_task_runner.py:52} INFO - Started process 1826 to run task
[2022-09-23 14:36:21,165] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'kag_to_gcs_dag', 'cleanup_list_task', 'manual__2022-09-23T14:31:12.806772+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/kag_to_gcs_dag.py', '--cfg-path', '/tmp/tmpsu1r67m9', '--error-file', '/tmp/tmpftp3y6au']
[2022-09-23 14:36:21,165] {standard_task_runner.py:77} INFO - Job 5: Subtask cleanup_list_task
[2022-09-23 14:36:21,207] {logging_mixin.py:109} INFO - Running <TaskInstance: kag_to_gcs_dag.cleanup_list_task manual__2022-09-23T14:31:12.806772+00:00 [running]> on host 31708a4d5ab1
[2022-09-23 14:36:21,250] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=kag_to_gcs_dag
AIRFLOW_CTX_TASK_ID=cleanup_list_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-23T14:31:12.806772+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-23T14:31:12.806772+00:00
[2022-09-23 14:36:21,271] {python.py:175} INFO - Done. Returned value was: None
[2022-09-23 14:36:21,281] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=kag_to_gcs_dag, task_id=cleanup_list_task, execution_date=20220923T143112, start_date=20220923T143621, end_date=20220923T143621
[2022-09-23 14:36:21,336] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-23 14:36:21,365] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
